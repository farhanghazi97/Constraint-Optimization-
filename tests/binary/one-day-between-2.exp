m1:mon 9am
m2:wed 9am
cost:0
