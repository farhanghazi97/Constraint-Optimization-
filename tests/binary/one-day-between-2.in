meeting, m1
meeting, m2
constraint, m1 one-day-between m2
domain, m2, wed, hard
