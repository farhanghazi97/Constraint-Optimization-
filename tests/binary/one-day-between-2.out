m1:mon 4pm
m2:wed 2pm
cost:0
